docs/usage/mnist.rst

```diff
@@ -48,28 +48,32 @@ Assess the model's accuracy.
 The accuracy likely won't be very good right now because we need to normalize
 the data first.
 
-Create an ``image.csv`` file which contains the names of the images to predict on.
+Create an ``image.csv`` file which contains the names of the images (with their extension .mnistpng) to predict on.
 
 .. literalinclude:: /../examples/MNIST/image_file.sh
 
-In this example, the ``image.csv`` file contains the names of the following images :-
+In this example, the ``image.csv`` file contains the names of the following images
 
-.. image:: /../examples/MNIST/image1.png
+.. image:: /../examples/MNIST/image1.mnistpng
     :width: 140px
     :height: 140px
 
-.. image:: /../examples/MNIST/image2.png
+.. image:: /../examples/MNIST/image2.mnistpng
     :width: 140px
     :height: 140px
 
-.. image:: /../examples/MNIST/image3.png
+.. image:: /../examples/MNIST/image3.mnistpng
```

Comment:
Let's not rename these files, instead seem my comment about testing. Leaving them as `.png` in the `examples/MNIST` folder will allow them to be displayed correctly by the documentation generator.

We should also add a note asking users to download each image and save them with their `.mnistpng` extention.

Reply:
I thought of not renaming them, but the docs were working completely fine with this change too.
Also I'll add the note.

Reply:
Don't worry about it if the docs looked fine! My bad.